lua/lazyvim/plugins/extras/lang/typst.lua

```diff
@@ -0,0 +1,60 @@
+return {
+  recommended = function()
+    return LazyVim.extras.wants({
+      ft = { "typst" },
+    })
+  end,
+
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = {
+      ensure_installed = { "typst" },
+    },
+  },
+
+  {
+    "neovim/nvim-lspconfig",
+    opts = {
+      servers = {
+        tinymist = {
```

Comment:
```suggestion
        tinymist = {
          single_file_support = true,
          root_dir = function()
            LazyVim.root.get()
          end,
```

I've been running into some issues with LSP not attaching if the the typst file isn't in a git repo. Enabling single file support fixes that, and works as normal otherwise. Actually, I'm not sure if `single_file_support = true` is needed because as far as I can tell LazyVim.root will always return the cwd if no other root is found.

There's more discussion about this at https://github.com/neovim/nvim-lspconfig/pull/3075